jerry-core/jrt/jrt.h

```diff
@@ -108,26 +108,14 @@ extern void __noreturn jerry_unimplemented (const char *, const char *, const ch
 #define JERRY_DDLOG(...) JERRY_LOG (2, __VA_ARGS__)
 #define JERRY_DDDLOG(...) JERRY_LOG (3, __VA_ARGS__)
 #else /* !JERRY_ENABLE_LOG */
-#define JERRY_DLOG(...) \
-  do \
-  { \
-    if (false) \
-    { \
-      jerry_ref_unused_variables (0, __VA_ARGS__); \
-    } \
-  } while (0)
-#define JERRY_DDLOG(...) JERRY_DLOG (__VA_ARGS__)
-#define JERRY_DDDLOG(...) JERRY_DLOG (__VA_ARGS__)
+#define JERRY_DLOG(...)
+#define JERRY_DDLOG(...)
+#define JERRY_DDDLOG(...)
 #endif /* !JERRY_ENABLE_LOG */
 
 #define JERRY_ERROR_MSG(...) jerry_port_errormsg (__VA_ARGS__)
 #define JERRY_WARNING_MSG(...) JERRY_ERROR_MSG (__VA_ARGS__)
 
-/**
- * Mark for unreachable points and unimplemented cases
- */
-template<typename... values> extern void jerry_ref_unused_variables (const values & ... unused);
```

Comment:
Couldn't we implement the same using macro with `__VA_ARGS__` and `(void) var_name;` statements?

Seems that its better to use `__attr_unused__` as least as possible, because the attribute could hide actual issues.